app/controllers/users/invitations_controller.rb

```diff
@@ -21,8 +21,7 @@ class Users::InvitationsController < Devise::InvitationsController
   def create
     authorize!(:manage, @vita_partners.find(invite_params[:vita_partner_id]))
     super do |invited_user|
-      # set default values
-      invited_user.update(role: invited_user.role || "agent")
+      invited_user.memberships.create(vita_partner_id: invite_params[:vita_partner_id])
```

Comment:
I would add a param for "role" to this, and allow certain users to be able to set a role for new users they invite.

Reply:
Allowing users to invite people to specific roles is a ticket in next week's sprint, so we were planning on deferring that update to that ticket.